target/linux/ath79/dts/ar9342_ubnt_acb-ac.dts

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "ubnt,acb-ac", "qca,ar9342";
```

Comment:
Please use "full" name, i.e ubnt,aircube-ac
Same for device/image name etc.

Reply:
thank you, kept common configuration as 'acb' and device full names have full 'aircube' in them now.